src/Ledger/GovernanceActions.lagda

```diff
@@ -24,7 +24,8 @@ open import Ledger.GovStructure
 
 module Ledger.GovernanceActions (gs : _) (open GovStructure gs) where
 
-2ℚ = 1ℚ Data.Rational.+ 1ℚ
+DENIED : ℚ
+DENIED = 1ℚ Data.Rational.+ 1ℚ
```

Comment:
I don't like the name `DENIED`. It suggests that you can't or shouldn't vote, but the opposite is true: we use this to make sure we never cut off voting early, no matter how much stake is in favour. Maybe we could put something like `vote*` in the table, to suggest there'll be voting but no number. Then this could be named `unenactable` or something.